lib/app/modules/home/views/add_task_bottom_sheet_new.dart

```diff
@@ -1,3 +1,526 @@
+// import 'dart:collection';
```

Comment:
Please remove this entire commented section. You should try to avoid adding irrelevant commented section in my opinion. It is not considered a good practice.